samples/dnn/models.yml

```diff
@@ -63,6 +63,19 @@ tiny-yolo-voc:
   classes: "object_detection_classes_pascal_voc.txt"
   sample: "object_detection"
 
+yolov8:
+  load_info:
+    url: "https://github.com/JasonLin1110/ultralytics_yolov8_onnx_model/yolov8net.onnx"
+    sha1: "67144099b4d8c40fa2d1f8cd17b2a6c6bb81ce90"
+  model: "yolov8net.onnx"
+  mean: [0, 0, 0]
+  scale: 0.00392
+  width: 640
+  height: 640
+  rgb: true
+  classes: "object_detection_classes_yolov8.txt"
```

Comment:
missed file

Reply:
here's our classes: "https://github.com/JasonLin1110/ultralytics_yolov8_onnx_model/blob/main/object_detection_classes_yolov8.txt", should we update new url to models.yml?

Reply:
This is identical to https://github.com/opencv/opencv/blob/4.x/samples/data/dnn/object_detection_classes_yolov4.txt

Reply:
We have updated url and classes in models.yml's yolov8 part, please check. Thank you!